R/model_options.R

```diff
@@ -83,7 +83,7 @@ build_json <- function(options_template, params) {
     glue::glue(options_template, .envir = param_env, .open = "<+", .close = "+>",
                .transformer = json_transformer),
     error = function(e) {
-      e$message <- t_("MODEL_OPTIONS_FAIL", list(message = message))
+      e$message <- t_("MODEL_OPTIONS_FAIL", list(message = e$message))
```

Comment:
This change implies that the previous behaviour was broken - could you add a quick test for it now it's fixed please?

Reply:
Yep - I've expanded a test we previously had written for this (it was only checking the first half of it though then matching on \w+) I've updated that so it checks it matches the entire string.